multicluster/build/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM gcr.io/distroless/static:nonroot as product
+WORKDIR /
+COPY ./bin/manager /
+USER 65532:65532
```

Comment:
when merging this to main in the future, please use a Dockerfile that doesn't assume that (the correct version of) Go is installed on the build machine and that the binary / binaries are built outside of docker.